src/nbl/video/debug/COpenGLDebugCallback.cpp

```diff
@@ -0,0 +1,45 @@
+#include "nbl/video/debug/COpenGLDebugCallback.h"
+
+using namespace nbl;
+using namespace nbl::video;
+
```

Comment:
@achalpandeyy take note of this for the Vulkan counterpart

Reply:
resolved